src/popover.coffee

```diff
@@ -234,8 +234,15 @@ Ember.Widgets.PopoverComponent.reopenClass
   rootElement: '.ember-application'
   hideAll: -> $(document).trigger('popover:hide')
 
-  popup: (options) ->
-    @hideAll()
+  ###*
+   * Shows a popup with options used for PopoverComponent.
+   * @param {Object} options All options used for PopoverComponent.
+   * @param {boolean} [hideOthers=true] Indicates if other popover should be hidden when a new one
+   *    is shown. By default, it's set to true.
+  ###
+  popup: (options, hideOthers = true) ->
```

Comment:
We actually used to pass hideOthers as an option, and had hideOthers as an attribute of the component. Billy thought this was misleading since hideOthers is only used when launching the popup, with the value of hideOthers in the component being ignored. We pass it as a parameter of `popup